docs/topics/snippets.md

```diff
@@ -0,0 +1,217 @@
+(snippets)=
+
+# Snippets
+
+Snippets are pieces of content which do not necessitate a full webpage to render. They could be used for making secondary content, such as headers, footers, and sidebars, editable in the Wagtail admin. Snippets are Django models which do not inherit the `Page` class and are thus not organised into the Wagtail tree. However, they can still be made editable by assigning panels and identifying the model as a snippet with the `register_snippet` class decorator.
+
+Snippets lack many of the features of pages, such as being orderable in the Wagtail admin or having a defined URL. Decide carefully if the content type you would want to build into a snippet might be more suited to a page.
+
+## Snippet Models
+
+Here's an example snippet model:
+
+```python
+from django.db import models
+
+from wagtail.admin.panels import FieldPanel
+from wagtail.snippets.models import register_snippet
+
+...
```

Comment:
```suggestion
# ...
```
Minor preference of mine - putting `...` in a python comment so the whole code can be copied / pasted without errors.

Reply:
I found some more places where this was not a comment, will commit that change.